lib-user: Certificate, change link label, center name, fix font color on dark mode print

mcbouslog opened the pull request:
## Package
- lib-user

## Describe your changes
- ~noticed the nice css pseudo element in the Dashboard, more performant and cleaner than...~
- ~in the Certificate, replaced the SVG with a css pseudo element to create the gradient line under the signature image~
- reverted the above noted changes, per https://github.com/zooniverse/front-end-monorepo/pull/6145#pullrequestreview-2139971135
- change link label from "Generate Certificate" to "Save Certificate"
- center align user name
- fix font color on print/save in dark mode, to consistent #5c5c5c with light mode

## How to Review
_Helpful explanations that will make your reviewer happy:_
- What user actions should my reviewer step through to review this PR?
  - compare existing line under signature: https://fe-root.preview.zooniverse.org/users/[your user login here]/stats/certificate
  - to these changes (app-root, add `?env=__` as needed): https://local.zooniverse.org:3000/users/[your user login here]/stats/certificate
- Which storybook stories should be reviewed?
  - the image is (known/intentionally) broken, but the line can still be seen/compared: http://localhost:6008/?path=/story/components-certificate--default 
- Are there plans for follow up PR’s to further fix this bug or develop this feature? no

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [ ] Tests are passing locally and on Github
- [ ] Documentation is up to date and changelog has been updated if appropriate
- [ ] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)
- [ ] FEM works in a mobile browser

## General UX
Example Staging Project: [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
- [ ] All pages of a FEM project load: Home Page, Classify Page, and About Pages
- [ ] Can submit a classification
- [ ] Can sign-in and sign-out
- [ ] The component is accessible
  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/assistive-technology/testing-steps/voiceover-mac.html)
  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - It is passing accessibility checks in the storybook

## Refactoring
- [ ] The PR creator has described the reason for refactoring
- [ ] The refactored component(s) continue to work as expected

shaunanoordin reviewed (dismissed):
## PR Review

This PR is a very small, non-functional UI update that changes how the decorative line below Laura's signature on the cert is created (from SVG to CSS). And how it looks, I guess.

Before this PR:
<img width="234" alt="image" src="https://github.com/zooniverse/front-end-monorepo/assets/13952701/368c5bfb-79a5-41ac-99c8-ed8c0b0d8480">

After this PR:
<img width="262" alt="image" src="https://github.com/zooniverse/front-end-monorepo/assets/13952701/9178ac2a-4425-4e4b-b830-c62916817c7a">

Code looks good.

### Dev Notes

However, I did notice that (on macOS + Chrome 126), the decorative line doesn't appear properly on the print preview. (i.e. after pressing Cmd+P or selecting File -> Print)

Before this PR:
<img width="434" alt="image" src="https://github.com/zooniverse/front-end-monorepo/assets/13952701/0e2e0762-d852-48cc-bee1-5aeb0a71af40">

After this PR:
<img width="434" alt="image" src="https://github.com/zooniverse/front-end-monorepo/assets/13952701/b5ce2965-1083-40e1-9f57-745cb77bccd6">

(Huh, I also only just noticed that on print, the display name isn't properly centre-aligned.)

### Status

Uh, I'm gonna approve this since it doesn't break anything per se, but please take a note of the quirk of the decorative line disappearing on print, and decide if that's OK for your purposes.

mcbouslog commented on packages/lib-user/src/components/Certificate/Certificate.js:
```diff
@@ -37,6 +38,11 @@ const PrintableBox = styled(Box)`
+
+    #certificate svg,
+    #certificate svg g {
+      fill: #00979d;
```
This keeps the Zooniverse logo the brand teal

mcbouslog commented:
@shaunanoordin - thank you for noting the decorative line doesn't show on print! I tried to get the css pseudo element to show on print without success. The svg shows on print, and is ok, so I've reverted the css pseudo element change back to the svg, centered the name, and fixed a font color issue when printing in dark mode. I've changed the PR name accordingly, updated the initial description. Since these changes have fundamentally changed I'll re-request a review, but no rush on this one.

shaunanoordin reviewed (approved):
## PR Review (Update)

This PR's updates are looking good!

Screenshot: darkeshard's certificate

<img width="442" alt="image" src="https://github.com/user-attachments/assets/87a97a84-c89b-4554-90e1-eefd81bffa68">

- Name is now centre-aligned
- Decorative line after Laura's signature is now visible on print.

I say this PR is good to go. 👍
